### ISSUE TITLE
Publish to Maven Local error

### ISSUE THREAD

**Issue by takase1121** [sedmelluq/lavaplayer#234]
```
> Task :node:publishMavenJavaPublicationToMavenLocal FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':node:publishMavenJavaPublicationToMavenLocal'.
> Failed to publish publication 'mavenJava' to repository 'mavenLocal'
   > Invalid publication 'mavenJava': artifact file does not exist: '/home/takase/lavaplayer/node/build/libs/node-1.2.50-boot.jar'

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s
20 actionable tasks: 9 executed, 11 up-to-date
```

This is what I get when I tried to run `./gradlew publishToMavenLocal`. I used `--debug` too, and I noticed that the `*-boot.jar`'s task was not executed. That's why it wasn't there. Did I do something wrong?

**Comment by s-frei** [comment]
You have to build the *boot.jar first on your own.

node -> Tasks -> build -> bootJar

**Comment by takase1121** [comment]
alright i will try it when I am free.

**Comment by s-frei** [comment]
That worked for me. Just ask if something doesn't work 😉

**Comment by robinfriedli** [comment]
I have the boot jar but `/node/build/libs/node-1.2.50.jar` is missing

**Comment by s-frei** [comment]
Then execute the **jar**  task next to **bootJar**.

**Comment by robinfriedli** [comment]
I did. That does not generated the required jar. I got it to work by removing the "sources" classifier from the sourceJar artifact in the node build.gradle and rerunning `gradle publishToMavenLocal`, which failed because of `Invalid publication 'mavenJava': multiple artifacts with the identical extension and classifier ('jar', 'null').` but generated the correct jar, then reverting the changes made to the build.gradle and rerunning the command again.
